.windsurf/rules/project-context-structure-and-formatting.md

```
@@ -52,6 +52,49 @@ description: "Concise description for SEO and navigation"
+**Protocols**. Now, you can deploy [Global Nodes](/docs/global-elastic-node) for Unichain Mainnet. See also [Unichain tooling](/docs/unichain-tooling) and a [tutorial](/docs/unichain-collecting-uniswapv4-eth-usdc-trades).
+
+<Button href="/changelog/chainstack-updates-may-30-2025">Read more</Button>
+</Update>
+```
+
+2. **Create file in `changelog/` directory**: Use format `chainstack-updates-may-30-2025.mdx`. Paste the same entry content (without the `<Update>` tags) from step 1.
+
+3. **Update `docs.json`**: In the `Release notes` section, add the newly created file name (without `.mdx`) between `changelog` and the previous release notes entries.
+
+Example:
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)